galaxy/importer/collection.py

```diff
@@ -144,6 +144,10 @@ def _load_contents(self, content_list):
             loader.lint()
             # content.scores = loader.score()
             self.log.info(' ')
+            self.log.info('===== IMPORTING {}{} ====='.format(
```

Comment:
missing a `": "` here between the content type and name, to restore the original behavior

Reply:
The colon is already in the variable `name`

Reply:
👍